addons/calendar/models/calendar_recurrence.py

```diff
@@ -345,10 +345,14 @@ def _rrule_parse(self, rule_str, date_start):
             data['end_type'] = 'forever'
         return data
 
+    def _get_lang_week_start(self):
+        lang = self.env['res.lang']._lang_get(self.env.user.lang)
+        week_start = int(lang.week_start)  # lang.week_start ranges from '1' to '7'
+        return week_start
+
```

Comment:
I would change this to return the week start as `rrule` wants it. To avoid the `-1` which comes out of nowhere in the code, twice.
```suggestion
    def _get_rrule_lang_week_start(self):
        lang = self.env['res.lang']._lang_get(self.env.user.lang)
        week_start = int(lang.week_start)  # lang.week_start ranges from '1' to '7'
        return week_start - 1 # rrule expects an int from 0 to 6

```

Reply:
Done :+1: